src/scripts/main.js

```diff
@@ -1 +1,66 @@
 'use strict';
+
+document.addEventListener('contextmenu', (e) => {
+  e.preventDefault();
+});
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', (e) => {
+    if (e.button === 0) {
+      resolve('First promise was resolved');
+    }
+  });
+
+  setTimeout(() => reject(new Error('First promise was rejected')), 3000);
+});
+
+const promise2 = new Promise((resolve) => {
+  document.addEventListener('mousedown', (e) => {
+    if (e.button === 0 || e.button === 2) {
```

Comment:
The second promise resolves on both left and right mouse button clicks. Ensure this behavior is intended, as it might conflict with the logic of the third promise, which requires both buttons to be clicked.